### PR TITLE
[DRFT-302] Make baselines table's headers sticky

### PR DESCRIPTION
**Pull request by johnsonm325**
Sticky headers have been applied to all tables other than the comparison
table in this PR. To see all tables:

- Open add to comparison modal and view systems and baselines tables.
- Go to Baselines in side bar and view baselines table there
- Click Create baseline button and view systems and baselines tables.
- Click on a baseline to view details table
- Click Associated systems tab on baseline details page.
- If there are associate systems, you can view sticky header on table. Otherwise, click Add associated system to add them (this table should also have a sticky header).

If there is not enough content in stage to scroll to see sticky headers,
you can always shrink the window to test.

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: src/SmartComponents/BaselinesPage/EditBaselinePage/EditBaseline/EditBaseline.js [hunk 1]
```diff
@@ -45,10 +45,10 @@ class EditBaseline extends Component {
...
                 data-ouia-component-type='PF4/TableHeaderRow'
                 data-ouia-component-id='edit-baseline-table-header-row'>
-                { baselinesWrite ? <th></th> : null }
-                <th className="edit-baseline-header"><div>Fact</div></th>
-                <th className="edit-baseline-header"><div>Value</div></th>
-                <th></th>
+                { baselinesWrite ? <th className="sticky-header"></th> : null }
+                <th className="edit-baseline-header sticky-header"><div>Fact</div></th>
+                <th className="edit-baseline-header sticky-header"><div>Value</div></th>
+                <th className="sticky-header"></th>
```
Comment (johnsonm325) [inline src/SmartComponents/BaselinesPage/EditBaselinePage/EditBaseline/EditBaseline.js:51]: If anyone can help me figure out how to add a gray border to the bottom of this sticky header, that would be great. No matter where I add it, it doesn't show up. To view this table, navigate to Drift -> Baselines and click on a baseline to view its facts. You will see the gray border until the sticky header goes into affect. I can demo if you're having a hard time reproducing.

Comment (bastilian) [reply src/SmartComponents/BaselinesPage/EditBaselinePage/EditBaseline/EditBaseline.js:51]: Have you tried `!important`?

Comment (johnsonm325) [reply src/SmartComponents/BaselinesPage/EditBaselinePage/EditBaseline/EditBaseline.js:51]: @bastilian didn't seem to make a difference.

**Review by leSamo — APPROVED** [review]
Works like a charm, thank you for providing detailed steps for testing.

**Pull request merged**